bedrock/mozorg/templates/mozorg/home/home.html

```diff
@@ -88,16 +88,16 @@
         <span class="toggle" role="button" aria-controls="nav-main-menu" aria-expanded="false" tabindex="0">{{_('Menu')}}</span>
         <ul id="nav-main-menu">
           <li class="first about-item">
-            <a data-link-type="about" data-element-location="nav click" href="{{ url('mozorg.about') }}">{{_('About')}}</a>
+            <a href="{{ url('mozorg.about') }}" data-link-type="nav" data-link-name="About">{{_('About')}}</a>
```

Comment:
I'll make the updates. I didn't see any failing tests when I ran `./manage.py test` in the terminal. Is that a concern?